how-to/use_terraform.rst

```diff
@@ -13,11 +13,11 @@ For this how-to you will need the following:
 
 - An identity provider that can be used to create OAuth2.0 client credentials.
 - Client credentials (``client_id`` and ``client_secret``) generated by the above identity provider.
-- A deployed JIMM configured to trust the identity provider. For instructions on 
-  how to deploy JIMM read :doc:`deploy_jimm`.
-- A Juju 3.5 controller added to JIMM that can be used to control your chosen cloud. For instructions
+- A deployed JAAS configured to trust the identity provider. For instructions on 
+  how to deploy JIMM read :doc:`../tutorial/deploy_jaas_microk8s`.
+- A Juju controller added to JIMM that can be used to control your chosen cloud. For instructions
   on how to add one read :doc:`add_controller`.
-- A Juju 3.5 client.
+- A Juju client.
```

Comment:
Why not mentioning Juju 3.5? Actually, this should be 3.5.1 or higher.

Reply:
Because then it sounds like you can't use Juju 3.6. We explain the minimum versions required to use JAAS in a separate doc.